Gds/src/fprime_gds/executables/tcpserver.py

```diff
@@ -32,7 +32,7 @@
 GUI_ids = []
 
 
-def signal_handler(signal, frame):
```

Comment:
This won't work (I don't think).  This will be passed a signal and a frame, you'll want to replace these with _, _